app/code/Magento/InventorySalesApi/Test/Api/IsProductSalableTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySalesApi\Test\Api;
+
+use Magento\TestFramework\TestCase\WebapiAbstract;
+use Magento\Framework\Webapi\Rest\Request;
+
+class IsProductSalableTest extends WebapiAbstract
```

Comment:
hmm looks like another gap, as I expected we already have web API test coverage for `IsProductSalable`

Reply:
just to say it once again - it's very cool, that you @phoenix128  decided to spent some part of your weekend on finalizing MSI for 2.3.0 release, as you found and filled some gaps we had.

Reply:
We did not have before. That is the class I added yesterday togheter with `\Magento\InventorySalesApi\Test\Api\IsProductSalableForRequestedQtyTest`